docs/_templates/layout.html

```diff
@@ -14,3 +14,13 @@
 <script src="{{ pathto('_static/js/viewer.min.js', 1) }}"></script>
 <script src="/assets/versions_raw.js"></script>
 {% endblock %}
+
+{% block docs_navbar %}
+{{ super() }}
+<div class="transition-banner container-fluid alert alert-info alert-dismissible fade show" role="alert">
+    <p>OpenVINO 2022.1 introduces a new version of OpenVINO API (API 2.0). For more information on the changes and transition steps, see the <a href="openvino_2_0_transition_guide.html">transition guide</a></p>
```

Comment:
```suggestion
    <p>Hey developers! We have introduced a new version of API (as known as API 2.0) since OpenVINO 2022.1. To learn about the updates and have an easier transition experience, check out our <a href="openvino_2_0_transition_guide.html">Transition Guide</a>.</p>
```

Reply:
I'm not sure what it will look like - just assuming that it's a banner on the top or a popup, so I think it would be nicer to use a friendly tone to not scare people away.

Reply:
@xu-yuan1 you can check what it looks like here (thanks Andrey):
http://openvino-docs.inn.intel.com/transition-banner/index.html
I'm not sure the tone should be as informal, as the banner is a short disclaimer. Maybe something like this?

<p>With OpenVINO 2022.1 we have introduced the new API 2.0. To learn about the changes and make your transition easier, check out our <a href="openvino_2_0_transition_guide.html">Transition Guide</a>.</p>

Reply:
Thanks guys! I checked the length in Andrey's link, so if we use Karol's more concise version and add "Hey developers!", it will probably fit well. I want to add this because it increases the possibility of clicking for me as a user. What do you think?
Besides, does the color have to light blue? do we want to use a brighter color (e.g. yellow, orange) to make it more highlighted?

Reply:
Folks, let's use the current wording and discuss possible improvements later.